pkgs/development/python-modules/nupd/default.nix

```
@@ -0,0 +1,76 @@
+{
+  lib,
+  buildPythonPackage,
+  pythonOlder,
+  fetchFromGitHub,
+
+  # build-system
+  setuptools,
+
+  # dependencies
+  aiohttp,
+  attrs,
+  frozendict,
+  inject,
+  loguru,
+  nbdb,
+  nix-prefetch-git,
+  nurl,
+  platformdirs,
+  typer,
+
+  # tests
+  pytestCheckHook,
+  aioresponses,
+  pytest-asyncio,
+  pytest-cov-stub,
+  pytest-mock,
+}:
+buildPythonPackage rec {
+  pname = "nixpkgs-updaters-library";
+  version = "0.1.0";
```

That tag seems to have vanished.

```suggestion
  version = "1.0.0";
```

---

Oops, don't know how that happened. The hash is correct though